Applications/spire/include/spire/time_and_sales/time_and_sales_window.hpp

```diff
@@ -11,6 +12,7 @@
 #include "spire/time_and_sales/time_and_sales_properties.hpp"
 #include "spire/time_and_sales/time_and_sales_window_model.hpp"
 #include "spire/ui/security_stack.hpp"
+#include "spire/ui/ui.hpp"
 #include "spire/ui/custom_qt_variants.hpp"
```

Comment:
Is this include needed here? I don't think you need to include custom_qt_variants in this file.